ci/prow/boskos/config.yaml

```diff
@@ -53,7 +53,7 @@ spec:
       terminationGracePeriodSeconds: 30
       containers:
       - name: boskos
-        image: gcr.io/k8s-testimages/boskos:v20190326-21e7f896f
+        image: gcr.io/k8s-testimages/boskos:v20190605-77876e8
```

Comment:
can we use an image a few days old instead of latest? We should use what k8s uses

Reply:
Yes, use what k8s is using.